routers/web/web.go

```diff
@@ -489,7 +487,7 @@ func registerWebRoutes(m *web.Router) {
 	m.Post("/-/markup", reqSignIn, web.Bind(structs.MarkupOption{}), misc.Markup)
 
 	m.Get("/-/web-theme/list", misc.WebThemeList)
-	m.Post("/-/web-theme/apply", optSignInIgnoreCsrf, misc.WebThemeApply)
+	m.Post("/-/web-theme/apply", misc.WebThemeApply)
```

Comment:
I have told you.

It NEED to "optSignIn"

Reply:
Ah yes, sorry.